Analyzer/plugins/Analyzer.cc

```diff
@@ -3794,6 +3851,44 @@ float Analyzer::combineProbs(float probOnTrackWMulti, int numRecHits) const {
   return probOnTrack;
 }
 
+
+//=============================================================
+//     Check if the GenIDs are for a HSCP
+//=============================================================
+bool Analyzer::isHSCPgenID(const reco::GenParticle& gen) {
```

Comment:
@kimsw9284 @DenkMybu this could be good for both of you

Reply:
thanks for these